src/main/java/Transcendent.java

```diff
@@ -0,0 +1,178 @@
+import java.util.Scanner;
+
+public class Transcendent {
+
+    public static void main(String[] args) {
+        Printer.printEntry();
+        List.init();
+        InputHandler.takeInput();
+        Printer.printExit();
+    }
+
+    private static class Printer {
+
+        private static void printEntry() {
+            Printer.printSeparator();
+            System.out.println("Greetings.");
+            System.out.println("I am Transcendent.");
+            System.out.println("What do you need assistance with?");
+            Printer.printSeparator();
+        }
+
+        private static void printSeparator() {
+            System.out.println("____________________________________________________________");
+        }
+
+        private static void printExit() {
+            Printer.printSeparator();
+            System.out.println("Let me know should you need assistance again.");
+            System.out.println("Farewell.");
+            Printer.printSeparator();
+        }
+
+        private static void printAddConfirm(String task) {
```

Comment:
is there any reason for adding the word confirm for adding and removing tasks? Could the naming be a bit more consistent overall